website/blog/2025/01-10/index.md

```diff
@@ -0,0 +1,268 @@
+---
+slug: crawlee-for-python-v05
+title: Crawlee for Python v0.5
+description: Announcing the Crawlee for Python v0.5 release.
+authors: [VladaD]
+---
+
+Crawlee for Python v0.5 is now available! This is our biggest release to this date, bringing new ported functionality from the [Crawlee for JavaScript](https://github.com/apify/crawlee), brand-new features that are exclusive to the Python library (for now), a new consolidated package structure, and a bunch of bug fixes and further improvements.
+
+## Getting started
+
+You can upgrade to the latest version straight from [PyPI](https://pypi.org/project/crawlee/):
+
+```shell
+pip install --upgrade crawlee
+```
+
+Check out the full changelog on our [website](https://crawlee.dev/python/docs/changelog#050-2025-01-02) to see all the details. If you are updating from an older version, make sure to follow our [Upgrading to v0.5](https://crawlee.dev/python/docs/upgrading/upgrading-to-v0x#upgrading-to-v05) guide for smooth upgrade.
+
+## New package structure
+
+We have introduced a new consolidated package structure. The goal is to streamline development experience, help you find the crawlers you are looking for faster, and improve the IDE's code suggestions while importing.
+
+### Crawlers
+
+We have grouped all crawler classes (and their corresponding crawling context classes) into a single sub-package called `crawlers`. Here is a quick example of how the imports have changed:
+
+```diff
+- from crawlee.beautifulsoup_crawler import BeautifulSoupCrawler, BeautifulSoupCrawlingContext
++ from crawlee.crawlers import BeautifulSoupCrawler, BeautifulSoupCrawlingContext
+```
+
+Example of import code suggestions in the IDE:
```

Comment:
I'd try to sell this more - "Look how you can see all the crawlers that we have, ain't that cool!"